velox/docs/functions/spark/window.rst

```diff
@@ -12,4 +12,13 @@ Value functions
 
 Returns the value at the specified offset from the beginning of the window. Offsets start at 1.
 The offset should be a positive int literal. If the offset is greater than the number of values
-in the window, null is returned. It is an error for the offset to be zero or negative.
\ No newline at end of file
+in the window, null is returned. It is an error for the offset to be zero or negative.
+
+Rank functions
+---------------
+
+.. function:: row_number() -> integer
+   :noindex:
+
+Returns a unique, sequential number to each row, starting with one, according to the ordering of
```

Comment:
Would you generate the docs and confirm they look good. It seems you might need some indentation here.

Reply:
Fixed.